packages/path-proxy/README.md

```diff
@@ -8,3 +8,27 @@ Proxies paths to the right SPA location or pass to different origin
 | --------------------- | --------------------------------------------- |
 | `npm test`            | Run tests.                                    |
 | `npm test -- --watch` | Run tests, and re-run them when files change. |
+
+### Pull requests and commit messages
+
+This repo follows Conventional Commits, a standard format for writing commit messages. Each commit message becomes an entry in [CHANGELOG.md](./CHANGELOG.md), and the commit messages are also used to determine what version bump to apply.
+
+Read more about [Conventional Commits](https://www.conventionalcommits.org) for a description and examples!
+
+If you are working on a pull request, don't worry about commit message format. Commit early and often.
+
+When your pull request is merged, "squash and merge" should be used, and a Conventional Commit message written at that point. In this way, your pull request will become a single commit in the master branch and one entry in the CHANGELOG will be created.
+
+## Configuration
+
+Configuration can be provided by CLI flags, environment variables, or a configuration file. Arguments are processed in that order, so CLI flags take precedence over environment variables, which take precedence over the configuration file.
+
+| Option          | Description                             | CLI             | Env                   | config.json  | Default                     |
+| --------------- | --------------------------------------- | --------------- | --------------------- | ------------ | --------------------------- |
+| **config file** | Where to find the config file.          | `--config-file` | `SPASHIP_CONFIG_FILE` | N/A          | none                        |
+| **webroot**     | Directory to extract/deploy SPAs.       | `--webroot`     | `SPASHIP_WEBROOT`     | `"webroot"`  | `/var/www`                  |
```

Comment:
@kunyan I'm not understanding why the path-proxy needs a webroot config option.  I don't think path-proxy ever accesses the webroot, does it?

Reply:
https://github.com/spaship/spaship/blob/8f5e9266e74792e1f885f371f67b80f551ac3875/packages/path-proxy/index.js#L9-L18

There has a function to read spa directories. it need know where the SPA stored.

Reply:
Ah!  Makes sense.  I forgot that path-proxy needs a list of SPA directory names.